spec/inferno/dsl/fhir_client_spec.rb

```diff
@@ -76,23 +103,69 @@ def test_session_id
     end
   end
 
+  describe '#body_to_path' do
+    before do
```

Comment:
When you say `let(:some_var) { 'foo' }`, you are setting up something that looks like a variable but isn't. The first time `some_var` is referred to in each test, it will be initialized from scratch with the value from its `let` block. If it isn't referred to, then it won't be initialized. This prevents you from initializing a bunch of stuff for every test even if the test doesn't need it.

So you don't need to call things from `let` blocks in `before` blocks unless they're doing something special which has to be done before the line which refers to them in the test is executed (like setting up a mock endpoint). If a test looks like this:
```
run(some_test)
expect some_mocked_endpoint.to_have_been_requested.once
```
then `some_mocked_endpoint` will have had to have been set up prior to that line of code, otherwise `run(some_test)` couldn't have hit the mocked endpoint because it didn't exist yet.